deploy/helm/sumologic/values.yaml

```diff
@@ -4503,3 +4503,43 @@ opentelemetry-operator:
 
   manager:
     env:
+
+pvc-cleaner:
```

Comment:
As per Helm's best practices, this should be camelcase. This will also solve the problem of referring to these values elsewhere. See: https://helm.sh/docs/chart_best_practices/values/#naming-conventions

I'm not sure why helm lint doesn't catch this.